### PR TITLE
PV2-1884 script to clean up Audit tables during submission

### PR DESCRIPTION
**Pull request by hardik-desai-kcl**

### CODE REVIEW

File: src/Scripts/Database/Audit tables clean up during submission.sql [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+print 'delete DataLockEventPriceEpisode finished ' + convert(varchar(150), SYSDATETIME())
+
+DELETE Payments2.DataLockEvent
+WHERE EventId IN (
+	SELECT EventId FROM #DatalocksToDelete
+)
+
+print 'delete DataLockEvent finished ' + convert(varchar(150), SYSDATETIME())
+
+rollback tran
```
Comment (tom-gough) [inline src/Scripts/Database/Audit tables clean up during submission.sql]: did you mean to leave the rollback in?

File: src/Scripts/Database/Audit tables clean up during submission.sql [hunk 2]
```diff
@@ -0,0 +1,139 @@
+begin tran
+
+print 'start ' + convert(varchar(150), SYSDATETIME())
+
+declare @collectionPeriod as tinyint = 8
```
Comment (tom-gough) [inline src/Scripts/Database/Audit tables clean up during submission.sql]: standardise capitalisation of keywords?

Comment (hardik-desai-kcl) [reply src/Scripts/Database/Audit tables clean up during submission.sql]: done

File: src/Scripts/Database/Audit tables clean up during submission.sql (hunk 1 above)
Comment (hardik-desai-kcl) [reply src/Scripts/Database/Audit tables clean up during submission.sql]: this is intentional so that testers can test this script

**Pull request merged**